src/App.vue

```diff
@@ -50,50 +51,24 @@ export default {
       return this.mergedRoute.page
     },
     messages() {
-      return this.$store.getters['traction/messages']
+      // Access the messages from the root store
+      const rootStore = useRootStore()
+      // Return the messages from the traction module
+      return rootStore.messages || {}
     },
     hasMessages() {
       return !!Object.keys(this.messages).length
     },
   },
   methods: {
     dismiss(messageIndex) {
-      this.$store.commit('traction/removeMessage', messageIndex)
+      const rootStore = useRootStore()
+      rootStore.removeMessage(messageIndex)
     },
     clearAlerts() {
-      this.$store.commit('traction/clearMessages')
+      const rootStore = useRootStore()
+      rootStore.clearMessages()
     },
   },
 }
 </script>
-
-<style>
-#app {
```

Comment:
This does change the default font slightly. Maybe these apps styles should be added to index.html?

Reply:
Do you know what needs to be done? Not too clear on the tailwind stuff.

Reply:
Looks like it was just the default text colour causing the difference, I have updated it in the tailwind css file.